src/Microsoft.AspNetCore.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -38,6 +38,8 @@ public GeneratedTagHelperContext()
             TagHelperOutputIsContentModifiedPropertyName = "IsContentModified";
             TagHelperOutputContentPropertyName = "Content";
             TagHelperOutputGetChildContentAsyncMethodName = "GetChildContentAsync";
+            TagHelperAttributeTypeName = "Microsoft.AspNetCore.Razor.TagHelpers.TagHelperAttribute";
+            HtmlStringTypeName = "Microsoft.AspNetCore.Mvc.Rendering.HtmlString";
```

Comment:
Ehh, we could but it'd be inconsistent with how non-complex unbound attributes are handled. Those values are handled via `Html.Raw` which wraps the value in an `HtmlString`.If we were to swap both this and the default impl of `Html.Raw` then it'd all be good.